flair/datasets/sequence_labeling.py

```diff
@@ -564,6 +564,112 @@ def __init__(
         )
 
 
+class BUSINESS_HUN(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "ner",
```

Comment:
This argument is currently unused. You can add it to the `super()` call below as `tag_to_bioes=tag_to_bioes` so that the `ColumnCorpus` can use the `tag_to_bioes` setting.